src/clarity-angular/wizard/wizard-button.spec.ts

```diff
@@ -909,6 +913,20 @@ export default function(): void {
                     fixture.detectChanges();
                     expect(buttonComponent.isHidden).toBe(false, "updates when set back");
                 });
+
+                it("binds to the host even with ngIf is applied", () => {
+                    // Test when buttons are not ready
+                    buttonHub.buttonsReady = false;
+                    expect(buttonComponent.isHidden).toBe(false, "should always be false if buttons aren't ready");
+                    // Test with buttons ready.
+                    buttonHub.buttonsReady = true;
+                    fixture.detectChanges();
+                    expect(buttonComponent.isHidden).toBe(false, "should be the default value");
+                    // Test setting to hide
+                    myTestComponent.hideBtn = true;
+                    fixture.detectChanges();
+                    expect(buttonComponent.isHidden).toBe(true, "should now be hidden");
+                });
             });
```

Comment:
Is there an equivalent test for `isDisabled` (i.e. testing for it in conjunction with `buttonsReady`'s value)?

Reply:
That's a good point. It might be a good idea to test for that and would be trivial to add the test here now as opposed to later if a problem is found.

Reply:
Added another test to be safe, though it doesn't suffer from being a host binding.